src/Security/Authentication/JwtBearer/src/JwtBearerHandler.cs

```diff
@@ -265,6 +265,15 @@ protected override async Task HandleChallengeAsync(AuthenticationProperties prop
             }
         }
 
+        protected override Task HandleForbiddenAsync(AuthenticationProperties properties)
+        {
+            var authenticationFailedContext = new AuthenticationFailedContext(Context, Scheme, Options)
+            {
+                Exception = new Exception("Forbidden")
+            };
+            Response.StatusCode = 403;
+            return Events.AuthorizationFailed(authenticationFailedContext);
+        }
```

Comment:
Empty line between methods.
```suggestion
        }
        
```

Reply:
Reviewed and updated as suggested.

Reply:
Corrected.